lib/ccf/archive.py

```diff
@@ -297,6 +297,24 @@ def available_diffusion_unproc_names(self, subject_info):
         name_list = self._get_scan_names_from_full_paths(dir_list)
         return name_list
 
+		
+    def running_status_dir_full_path(self, subject_info):
+        """
+        Full path to the unprocessed diffusion data resource directory
```

Comment:
Change comment to "Full path to the running status resource directory"

Reply:
changed